specification/cognitiveservices/data-plane/Language/preview/2021-07-15-preview/questionanswering.json

```
@@ -346,25 +347,43 @@
       "additionalProperties": false,
       "properties": {
         "metadata": {
-          "type": "object",
-          "additionalProperties": {
-            "type": "string",
-            "minLength": 0,
-            "maxLength": 500
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/MetadataRecord"
           }
         },
-        "compoundOperation": {
+        "logicalOperation": {
           "type": "string",
-          "$ref": "#/definitions/CompoundOperationKind",
+          "description": "Operation used to join metadata filters.",
+          "$ref": "#/definitions/LogicalOperationKind",
           "default": "AND"
         }
       }
     },
-    "CompoundOperationKind": {
+    "MetadataRecord": {
+      "type": "object",
+      "description": "Object to provide the key value pair for each metadata.",
+      "additionalProperties": false,
+      "required": [
+        "key",
+        "value"
+      ],
+      "properties": {
+        "key": {
+          "type": "string",
+          "description": "Metadata Key from Metadata dictionary used in the QnA."
+        },
+        "value": {
+          "type": "string",
+          "description": "Metadata Value from Metadata dictionary used in the QnA."
+        }
+      }
+    },
+    "LogicalOperationKind": {
       "type": "string",
-      "description": "(Optional) Set to 'OR' for joining metadata using 'OR' operation.",
+      "description": "Set to 'OR'/'AND' for using corresponding logical operation.",
```

It reads and localizes better if you write:

```suggestion
      "description": "Set to 'OR' or 'AND' for using corresponding logical operation.",
```